wiki/Game_Modifiers/en.md

```diff
@@ -1,306 +1,531 @@
+![Game mods screen][GM]
+
+A **game modifier** is one of several optional modifications to gameplay the player can select. Some modifiers (mods) lower difficulty in various ways but assign a score penalty by multiplying the final score with a modifier less than 1.00. Other mods increase difficulty to provide a greater challenge and also reward the player with a modifier greater than 1.00 to the final score. Players who use the Hidden and Flashlight mods will also gain the possibility to attain Silver S and Silver SS grades. Finally, some mods can alter the gameplay experience without affecting difficulty (such as the Relax mod, which the player only need to move the cursor to the hit objects).
 
 Game modifiers can be accessed by pressing `F1` at song select, or by clicking the `Mods` button near the lower left of the screen.
 
-Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. *osu!* used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x.
+![Song Selection's Toolbox][Toolbox]
 
-![Game mods screen](img/Soloplay8.png "Game mods screen")
+Note that when multiple mods are applied, the product of the score multipliers is given instead of the sum of them. For example, when Hidden and Flashlight are applied at the same time, they give 1.1872x because (1.06 x 1.12) = 1.1872, rather than (1.06 + 1.12 - 1.00) = 1.18. _osu!_ used to round this number down to the nearest hundredth (0.01), so Hidden and Flashlight together would appear to give a 1.18x multiplier, denying the supposed number round up. Thankfully, a patch corrected this, and now Hidden + Flashlight rounds correctly to 1.19x.
 
-Difficulty Reduction Mods
--------------------------
+## Difficulty Reduction Mods
 
 These mods make the game easier at the cost of decreasing the score obtained.
 
 ### Easy
 
-![Easy mod icon](/wiki/shared/Easy.png "Easy mod icon")
+![Easy (EZ) mod icon][EZ]
 
-**Score multiplier**: 0.5
+> Score multiplier: 0.50
+> Shortcut key: `Q`
 
-**Description**: Reduces overall difficulty - larger circles, more forgiving HP drain, less accuracy required.
+Easy mod halves all difficulty settings of the beatmap selected at a cost of decreasing the score modifier by half.
```

Comment:
Simplified to

Easy mod halves all difficulty settings from the beatmap selected.